README.md

```diff
@@ -31,9 +31,31 @@ mix deps.get
 
 In your `config.exs` add your server-side sdk key from `step 3` above.
 ```Elixir
-config :cfclient, api_key: "YOUR_API_KEY"
+config :cfclient, api_key: System.get_env("HARNESS_API_KEY")
```

Comment:
For the sake of consistency, we should advise the env variable to be `FF_API_KEY` like the other examples.

Reply:
Thanks, changed